miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/file/MiniAppFileChooser.kt

```diff
@@ -58,6 +70,32 @@ class MiniAppFileChooserDefault(var requestCode: Int) : MiniAppFileChooser {
         return true
     }
 
+    /**
+     * Validation Utility for MimeTypes.
+     * @param mimeTypes The Mimetypes needs to convert to valid types.
+     */
+    @VisibleForTesting
+    internal fun extractValidMimeTypes(mimeTypes: Array<String>): List<String> {
+        val results: MutableList<String> = ArrayList()
```

Comment:
I think you don't need to create a temporary list here. You could instead use `mapNotNull`, for example:

```
mimeTypes.mapNotNull { mime -> 
  if ( // something //) {
    mime
  } else {
    null
  }
}
```